resources/views/components/fields/details/belongs-to.blade.php

```diff
@@ -0,0 +1,7 @@
+@props(['field', 'item' => null])
+
+<x-nebula::form-row :field="$field">
+
+    <p class="text-sm">{{ $field->resolveBelongsTo($item) }}</p>
```

Comment:
One idea might be making this a link to the related resource, if it exists as a resource in Nebula.

Reply:
This is done via the `BelongsTo` field, eg: `BelongsTo::make('Users', 'user_id')->resource(UserResource::class);`

Reply:
Could also implement an auto resolver like other classes do, but this will work for now